Header header
uint8 Type_DualSense = 0
uint8 Type_XBox360_Microsoft = 1
uint8 Type_XBox360_Logitech = 2
uint8 controllerType
float64[] axisVals
int32[] buttonVals
int32[] hatVals

